### PR TITLE
Add note about adding new images

### PR DESCRIPTION
**Pull request by sunnylqm**

### CODE REVIEW

**Comment by satya164** [comment]
I don't think it's the

**Comment by satya164** [comment]
I don't think it's the case. I can add new images instead without restarting the packager just fine.

**Comment by sunnylqm** [comment]
Well, then maybe it's another windows only case

**Comment by satya164** [comment]
@sunnylqm Most likely. Since Windows doesn't use watchman. Can you mention Windows in the note?

File: docs/Images.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ And `button.js` code contains
 
 Packager will bundle and serve the image corresponding to device's screen density, e.g. on iPhone 5s `check@2x.png` will be used, on Nexus 5 – `check@3x.png`. If there is no image matching the screen density, the closest best option will be selected.
 
+Note that if a packager is running on Windows when you are adding new images to your project, then you need to restart the packager in order to require them.
```
Comment (satya164) [inline docs/Images.md]:
```
__NOTE__: On Windows, you might need to restart the packager if you add new images to your project.
```

**Comment by frantic** [comment]
I'd rather have us investigate why. Is there an open issue for this? Does it only fail for images or JS files as well?

**Comment by sunnylqm** [comment]
AFAIK，it fails on every new images/modules under windows.

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Comment by satya164** [comment]
@frantic Opened an issue for it - https://github.com/facebook/react-native/issues/6318

**Pull request closed without merging**